src/Helpers/index.test.js

```diff
@@ -100,3 +104,63 @@ describe('removeGeoJSON', () => {
         expect(map.removeSource.called).to.equal(true);
     });
 });
+
+describe('convertRadiusUnit', () => {
+    beforeEach(() => {
+        sinon.spy(console, 'error');
+        sinon.spy(console, 'warn');
+    });
+
+    afterEach(() => {
+        global.console.error.restore();
+        global.console.warn.restore();
+    });
+
+    it('should display error "The radius given is not a number" when no radius is given', () => {
+        convertRadiusUnit();
+        expect(global.console.error.called).to.equal(true);
+    });
+
+    it('should display error "The radius given is not a number" when the radius given isNaN', () => {
+        convertRadiusUnit('NotANumber');
+        expect(global.console.error.called).to.equal(true);
```

Comment:
Both previous tests check the same output, can you use `calledWith` for checking accurate debug message ?